rest_framework/renderers.py

```diff
@@ -319,9 +320,14 @@ def render_field(self, field, parent_style):
             style['template_pack'] = parent_style.get('template_pack', self.template_pack)
         style['renderer'] = self
 
+        # If we are rendering a JSON Field we want to convert Python literals to Javascript literals
+        if issubclass(field._proxy_class, JSONField):
```

Comment:
This was the simplest solution but I'm not sure I like putting field specific logic into the generic `render_field` function - some considerations would be to allow a field to provide it's own `render_field` method which is shared mixin, or some other form which allows the fields to render a value transformer to be called before `get_rendered_html_form` returns.